tests/unit/staff/views/test_job_requests.py

```diff
@@ -169,13 +169,12 @@ def test_jobrequestlist_search_using_fullname(rf, core_developer):
     user = UserFactory(fullname="Ben Goldacre")
     job_request = JobRequestFactory(created_by=user)
 
-    request = rf.get("/?q=ben")
+    request = rf.get("/?q=ben goldacre")
```

Comment:
```suggestion
    request = rf.get("/?q=ben+goldacre")
```

Spaces get converted to a `+` for this query arg 🙂

Reply:
Ah, cool. Thank you. I'd assumed they were being converted, but wasn't sure what to and that seemed to work. I've updated it now